tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectTest.java

```diff
@@ -371,7 +371,7 @@ private static void whenProjectFullCleanUsingContextMenu() throws IOException
 		{
 			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "Project Full Clean");
 			ProjectTestOperations.joinJobByName(Messages.ProjectFullCleanCommandHandler_RunningFullcleanJobName);
-			ProjectTestOperations.waitForProjectClean(bot);
+			ProjectTestOperations.findInConsole(bot, "Espressif IDF Tools Console", "Done");
```

Comment:
@AndriiFilippov generally the when*** methods should not contain any verifications that can cause failures if we look deeper into this findInConsole method it can throw exception if it doesnot find this Done and in reality the script may have successfully executed. You are already verifying the project clean operations in the assertions so this is not required

Reply:
@AndriiFilippov please see this as well in next commit